IBPSA/Fluid/Storage/BaseClasses/PartialTwoPortInterface.mo

```diff
@@ -0,0 +1,113 @@
+    annotation (Placement(transformation(extent={{-10,90},{10,110}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b(
+    redeclare final package Medium = Medium,
+    m_flow(max=if allowFlowReversal then +Modelica.Constants.inf else 0),
+    p(start=Medium.p_default),
+    h_outflow(start = Medium.h_default, nominal = Medium.h_default))
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{10,-110},{-10,-90}})));
+
+  parameter Modelica.Units.SI.MassFlowRate m_flow_nominal
+    "Nominal mass flow rate" annotation (Dialog(group="Nominal condition"));
+  parameter Modelica.Units.SI.MassFlowRate m_flow_small(min=0) = 1E-4*abs(
+    m_flow_nominal) "Small mass flow rate for regularization of zero flow"
+    annotation (Dialog(tab="Advanced"));
+  // Diagnostics
+   parameter Boolean show_T = false
+    "= true, if actual temperature at port is computed"
+    annotation (
+      Dialog(tab="Advanced", group="Diagnostics"),
+      HideResult=true);
+
+  Modelica.Units.SI.MassFlowRate m_flow(start=_m_flow_start) = port_a.m_flow
+    "Mass flow rate from port_a to port_b (m_flow > 0 is design flow direction)";
+
+  Modelica.Units.SI.PressureDifference dp(
+    start=_dp_start,
+    displayUnit="Pa") = port_a.p - port_b.p
+    "Pressure difference between port_a and port_b";
+
+  Medium.ThermodynamicState sta_a=
+    if allowFlowReversal then
+      Medium.setState_phX(port_a.p,
+                          noEvent(actualStream(port_a.h_outflow)),
+                          noEvent(actualStream(port_a.Xi_outflow)))
+    else
+      Medium.setState_phX(port_a.p,
+                          noEvent(inStream(port_a.h_outflow)),
+                          noEvent(inStream(port_a.Xi_outflow)))
+      if show_T "Medium properties in port_a";
+
+  Medium.ThermodynamicState sta_b=
+    if allowFlowReversal then
+      Medium.setState_phX(port_b.p,
+                          noEvent(actualStream(port_b.h_outflow)),
+                          noEvent(actualStream(port_b.Xi_outflow)))
+    else
+      Medium.setState_phX(port_b.p,
+                          noEvent(port_b.h_outflow),
+                          noEvent(port_b.Xi_outflow))
+       if show_T "Medium properties in port_b";
+
+protected
+  final parameter Modelica.Units.SI.MassFlowRate _m_flow_start=0
+    "Start value for m_flow, used to avoid a warning if not set in m_flow, and to avoid m_flow.start in parameter window";
+  final parameter Modelica.Units.SI.PressureDifference _dp_start(displayUnit=
+        "Pa") = 0
+    "Start value for dp, used to avoid a warning if not set in dp, and to avoid dp.start in parameter window";
+
+  annotation (
+    preferredView="info",
+    Documentation(info="<html>
+<p>
+This partial class implements the same functionality as 
+<a href=\"modelica://IBPSA.Fluid.Interfaces.StaticTwoPortHeatMassExchanger\">
+IBPSA.Fluid.Interfaces.StaticTwoPortHeatMassExchanger</a>,
```

Comment:
I don't get why the last version extends `PartialTwoPort` and now you duplicated `StaticTwoPortHeatMassExchanger`

Reply:
I don't understand your comment. The implementation in this PR implements `PartialTwoPortInterface` and does not duplicate `StaticTwoPortHeatMassExchanger`. 

My new version of `PartialTwoPortInterface` implements however also `PartialTwoPort` directly inside it, as I did not want to have multiple layers of inheritance and never use all layers by themselves only.

Reply:
It was basically the same comment as Jelger down below, I just confused `PartiaTwoPortInferface` with `PartiaTwoPort`.